Tasks/DockerV2/dockerpush.ts

```diff
@@ -65,7 +65,8 @@ export function run(connection: ContainerConnection, outputUpdate: (data: string
     let commandArguments = isBuildAndPushCommand ? "" : dockerCommandUtils.getCommandArguments(tl.getInput("arguments", false));
 
     // get tags input
-    let tags = tl.getDelimitedInput("tags", "\n");
+    let delimiter = [",","\n"];
+    let tags = utils.getDelimitedInput(tl.getInput("tags"), delimiter);
```

Comment:
You could just run `tl.getInput("tags").split(/[\n,]+/)` instead.

Reply:
My comment in utils.ts holds true here as well. Hence using a new method.

Reply:
You could edit the function in utils to do the same. you could simply run `filter` on the array to remove the empty ones. What can be solved in a single line of code doesn't really justify this approx 20 line code. 

What do you think?